logging/runtime/pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.quarkiverse.googlecloudservices</groupId>
+        <artifactId>quarkus-google-cloud-logging-parent</artifactId>
+        <version>1.1.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-google-cloud-logging</artifactId>
+    <name>Quarkus - Google Cloud Services - Logging - Runtime</name>
+    <description>Use Google Cloud operations logging</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkiverse.googlecloudservices</groupId>
+            <artifactId>quarkus-google-cloud-common</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkiverse.googlecloudservices</groupId>
+            <artifactId>quarkus-google-cloud-common-grpc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-grpc-common</artifactId>
+        </dependency>
```

Comment:
Should not be needed as you already include `quarkus-google-cloud-common-grpc`.

Reply:
Check.

Reply:
Actually, I get this if I try to remove it: 

```
[ERROR] Quarkus Extension Dependency Verification Error
[ERROR] The deployment artifact io.quarkiverse.googlecloudservices:quarkus-google-cloud-logging-deployment::jar depends on the following Quarkus extension deployment artifacts whose corresponding runtime artifacts were not found among the dependencies of io.quarkiverse.googlecloudservices:quarkus-google-cloud-logging:jar:1.1.0-SNAPSHOT:
[ERROR]  io.quarkiverse.googlecloudservices:quarkus-google-cloud-logging-deployment:jar:1.1.0-SNAPSHOT
[ERROR] *  io.quarkus:quarkus-grpc-common-deployment:jar:2.7.0.Fina
```

Reply:
It's the other way around ;)
You should remove this dep and add the `io.quarkiverse.googlecloudservices:quarkus-google-cloud-common-grpc-deployment` dependency to the deployment module.

Reply:
Aaaand... Where can I find this `quarkus-google-cloud-common-grpc-deployment` thingy? If I try to add it with version `${project.version}` I get an artifact not found error. It feels like I'm missing something :-D

Reply:
I was misleading, you do need quarkus-grpc-common on your runtime module and quarkus-grpc-common-deployment on you deployment module.

The `quarkus-google-cloud-common-grpc` is a runtime only dependency for specific Google Cloud gRPC stuff (native support).